### ISSUE TITLE
Use EpiAware approach to running tests

### ISSUE THREAD

**Issue by seabbs** [EpiAware/PrimaryCensored.jl#27]
See here: https://github.com/CDCgov/Rt-without-renewal/blob/main/EpiAware/test/runtests.jl

This integrates unit testing with vscode testing support. Also see the test files where we shouldn't need to load Test (or any other libraries). 

We have been using `TestEnv.jl` to run tests interactively and it seems to work well

**Comment by SamuelBrand1** [comment]
This seems to have been implemented? Close this?

**Comment by seabbs** [comment]
I don't think it has. We aren't using the testitem runner macros and last time I checked there were still using calls?

**Comment by seabbs** [comment]
I think we just need to get rid of these: https://github.com/EpiAware/PrimaryCensored.jl/blob/94f012d8cf82bee554a3487c6e6e244dc8a67cab/test/PrimaryCensoredDist.jl#L1

**Comment by SamuelBrand1** [comment]
> I don't think it has. We aren't using the testitem runner macros and last time I checked there were still using calls?

Yeah we are? 
e.g

https://github.com/EpiAware/PrimaryCensored.jl/blob/94f012d8cf82bee554a3487c6e6e244dc8a67cab/test/WithinIntervalCensored.jl#L2
